### PR TITLE
Improve extension description and README

### PR DESCRIPTION
**Pull request by flodiebold**

### CODE REVIEW

File: editors/code/package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
     "name": "rust-analyzer",
     "displayName": "rust-analyzer",
-    "description": "A language server for the rust programming language",
+    "description": "The future of Rust support in VSCode.",
```
Comment (lnicola) [inline editors/code/package.json:4]: Also the present hopefully 😅,

**Review by lnicola — APPROVED** [review]

File: editors/code/package.json (hunk 1 above)
Comment (flodiebold) [reply editors/code/package.json:4]: Yeah I don't know, maybe someone has a nicer description :smile: I guess we could also go for something more 'boring', but I felt this helps a bit more to differentiate this extension from `rust-lang.rust`

Comment (lnicola) [reply editors/code/package.json:4]: The New and Improved Rust extension!

Comment (matklad) [reply editors/code/package.json:4]: I'd just say "Rust language support for Visual Studio Code". And the rust-lang.rust one should say "use the other one!"

File: editors/code/package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
     "preview": true,
     "private": true,
     "icon": "icon.png",
     "version": "0.4.0-dev",
```
Comment (matklad) [inline editors/code/package.json:8]: We probably want to fix the version generating code?

File: editors/code/package.json [hunk 3]
```diff
@@ -1,15 +1,15 @@
 {
     "name": "rust-analyzer",
     "displayName": "rust-analyzer",
-    "description": "A language server for the rust programming language",
+    "description": "The future of Rust support in VSCode.",
     "preview": true,
```
Comment (jonas-schievink) [inline editors/code/package.json:5]: We might as well remove the preview tag while we're at it

```suggestion
```

**Pull request merged**